lib/trento/application/integration/discovery/policies/host_policy.ex

```diff
@@ -95,6 +140,52 @@ defmodule Trento.Integration.Discovery.HostPolicy do
            installation_source: installation_source
          })
 
+  defp build_update_saptune_command(agent_id, nil, package_version, saptune_installed),
+    do:
+      UpdateSaptuneStatus.new(%{
+        host_id: agent_id,
+        saptune_installed: saptune_installed,
+        package_version: package_version,
+        status: nil
+      })
+
+  defp build_update_saptune_command(
+         agent_id,
+         %SaptuneDiscoveryPayload{
+           package_version: package_version,
+           configured_version: configured_version,
+           tuning_state: tuning_state,
+           services: services,
+           notes_enabled_by_solution: notes_enabled_by_solution,
+           notes_applied_by_solution: notes_applied_by_solution,
+           notes_enabled_additionally: notes_enabled_additionally,
+           solution_enabled: solution_enabled,
+           solution_applied: solution_applied,
+           notes_enabled: notes_enabled,
+           notes_applied: notes_applied,
+           staging: staging
+         },
+         package_version,
+         saptune_installed
+       ) do
+    UpdateSaptuneStatus.new(%{
+      host_id: agent_id,
+      package_version: package_version,
+      saptune_installed: saptune_installed,
+      status: %{
+        package_version: package_version,
+        configured_version: configured_version,
+        tuning_state: tuning_state,
+        services: format_saptune_services_list(services),
+        enabled_notes: format_saptune_notes(notes_enabled_additionally, notes_enabled),
```

Comment:
Incorrect order in `enabled_notes` and `applied_notes`.
They should `(notes_enabled, notes_enabled_additionally)` (the additional notes as 2nd argument)

Reply:
yes, nice catch! done